.gitignore

```diff
@@ -4,3 +4,6 @@ Cargo.lock
 
 # IDE files
 .idea
+
+# 3ds files
+*.cia
```

Comment:
We haven't standardized anything like this. We may do something similar with `cargo-3ds` in the future, but it will most likely end up in the `target` directory for easy integration with Cargo. Also, look at the review comment.

Reply:
(Look at the review comment about the `.rsf` file)